### PR TITLE
Add PSGI support

### PR DESCRIPTION
**Pull request by ppisar**
This resolves [https://github.com/openshift/sti-perl/issues/58]. It requires the augmented test/run from [https://github.com/openshift/sti-perl/pull/60] pull request.

### CODE REVIEW

File: 5.20/README.md [hunk 1]
```diff
@@ -58,6 +58,11 @@ Repository organization
             `cpanfile` file, in which the user specifies the modules and their versions.
             An example of a [cpanfile](https://github.com/openshift/sti-perl/blob/master/5.20/test/sample-test-app/cpanfile) is available within our test application.
 
+            All files with cgi and pl extension are handled by mod_perl. If
```
Comment (mfojtik) [inline 5.20/README.md]: "All files with `.cgi` and `.pl` extension" reads better

**Comment by mfojtik** [comment]
small nit, LGTM

File: 5.20/README.md (hunk 1 above)
Comment (ppisar) [inline 5.20/README.md]: On Tue, Nov 10, 2015 at 05:00:36AM -0800, Michal Fojtik wrote:
> "All files with `.cgi` and `.pl` extension" reads better
> 
I changes the text as you suggested (although I think extension presume a dot
delimiter, with dot it's a suffix) and rebased the branch.

**Comment by mfojtik** [comment]
@ppisar which PR you want to merge first :-) I think you will have to rebase the other one.

**Comment by ppisar** [comment]
On Thu, Nov 12, 2015 at 01:32:14AM -0800, Michal Fojtik wrote:
> @ppisar which PR you want to merge first :-) I think you will have to rebase
> the other one.
> 
I rebased and pushed both of them. The PSGI thing (PR #60) is built on top of
the extlib thing (PR #61) as it was previously.

So first is apply PR #60:

654099c2536dcc8703ee0d701933b7b0932a09fd
a0d33ab9e1b2332b07f0f9558d15bdf1012ce0eb

and then PR #61:

fc0eb0e0b0e98651ca23b8f998d1531012c803e1
de17b48b6ebf4080bad29a77f849e5a9a9d8ef69

Maybe GitHub cannot comprehend it. I can file a fresh new pull request, if it
help.

-- Petr

**Comment by ppisar** [comment]
And maybe you got bitten by the silly GitHub's merge commits that prevents from fast-forwarding consequential patchsets (pull request in GH langugage). After applying previous one an artificial merge commit is inserted and that effectively diverts from next one commit line.

Should I rebase this pull request again against now-master? I can see previous robot comment "This branch is up-to-date with the base branch".

**Comment by mfojtik** [comment]
@ppisar I think rebase and force push should get rid of the first commit and prevent merge commit :-) I would prefer that if it is possible.

**Comment by ppisar** [comment]
Done. Try it again.

**Comment by mfojtik** [comment]
Merged. Thanks!

**Pull request merged**